src/components/structures/RoomView.js

```diff
@@ -1154,6 +1184,21 @@ module.exports = React.createClass({
                         </div>
                     );
                 }
+                else if (this.tabComplete.isTabCompleting()) {
+                    var TabCompleteBar = sdk.getComponent('rooms.TabCompleteBar');
+                    statusBar = (
+                        <div className="mx_RoomView_tabCompleteBar">
+                            <div className="mx_RoomView_tabCompleteImage">...</div>
```

Comment:
As @ara4n pointed out, this is a literal `...` for display on the UI. It's confusingly been given a suffix of `Image` though on the class name which I find unclear.